### PR TITLE
Patch 0.8.1

### PR DESCRIPTION
**Pull request by bms63**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [ ] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/main/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html#deprecation-1)?
- [ ] Update to all relevant roxygen headers and examples 
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ ] Address any updates needed for vignettes and/or templates
- [ ] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [ ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [ ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [ ] Link the issue in the Development Section on the right hand side.
- [ ] Address all merge conflicts and resolve appropriately 
- [ ] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by bms63** [comment]
Hi all,  This will be `admiral 0.8.1` that addresses the hotfix for the bug identified by Ross and a few cosmetic changes for the website and readme

**Review by bundfussr — APPROVED** [review]

**Comment by bms63** [comment]
@thomas-neitmann  Hi Thomas.  If this looks good to you, then it can be tarred up and submitted to CRAN.

File: vignettes/contribution_model.Rmd [hunk 1]
```diff
@@ -27,7 +27,12 @@ Our contribution model is still in a testing phase so we advise you to contact u
 # Basics
  * For each new contribution, the user creates an issue on the issue tab on [GitHub](https://github.com/pharmaverse/admiral/issues) to put it in our backlog. 
  
- * Familiarize yourself with our [programming strategy](programming_strategy.html), guidance for [GitHub usage](git_usage.html) and [unit testing](unit_test_guidance.html).
+ * Familiarize yourself with our [programming
+ strategy](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html),
+ guidance for [GitHub
+ usage](https://pharmaverse.github.io/admiraldev/main/articles/git_usage.html)
+ and [unit
+ testing](https://pharmaverse.github.io/admiraldev/main/articles/unit_test_guidance.html).
```
Comment (bms63) [inline vignettes/contribution_model.Rmd:35]:
```suggestion
 * Familiarize yourself with our [programming
 strategy](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html),
 guidance for [GitHub
 usage](https://pharmaverse.github.io/admiraldev/devel/articles/git_usage.html)
 and [unit
 testing](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html).
```

File: vignettes/contribution_model.Rmd [hunk 2]
```diff
@@ -44,7 +49,9 @@ Type 1 Contribution with Code:
 
  * First, the user creates an issue or comments on an existing issue to notify that they’d like to contribute code.
  
- * Follow our [development process](development_process.html) step-by-step guide.
+ * Follow our [development
+ process](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html)
+ step-by-step guide.
```
Comment (bms63) [inline vignettes/contribution_model.Rmd:54]:
```suggestion
 * Follow our [development
 process](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html)
 step-by-step guide.
```

File: vignettes/faq.Rmd [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+##### How much of the **ADaM IG is covered by admiral?**
+
+ * ADaM  IG is a standard framework without a specific number of datasets or variables, so it cannot be used as a specific baseline to answer that question.
+ * We will provide guidance for each ADaM  dataset structure (ADSL, OCCDS and BDS) that will highlight which functionality `{admiral}` covers. (see [In **which order** does a user need to execute the functions?])
+ * The guidance will also highlight the gaps to be filled by the user (e.g. timing, ranges).
+ * For standard ADaM datasets (ADAE, ADCM, ...) we can provide an estimated coverage based on early adopters Roche/GSK ADaM implementation
+
+##### Will there be a user/**contribution** guide?
+
+ * Our [programming strategy](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html) serves as a framework
```
Comment (bms63) [inline vignettes/faq.Rmd:119]:
```suggestion
 * Our [programming strategy](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html) serves as a framework
```

File: vignettes/faq.Rmd [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+ * There might be equally suited programming languages out there - however at some stage we had to make a decision :)
+ 
+##### Why do we use a **certain R version and package versions** for development?
+
+* The choice of R Version is not set in stone.  However, a common development environment is important to establish when working across multiple companies and multiple developers. We currently work in R Version 3.6.3, but that will change as we move forward with `{admiral}`. This need for a common development environment also carries over for our choice of package versions.  
+* GitHub allows us through the Actions/Workflows to test `{admiral}` under several versions of R as well as several versions of dependent R packages needed for `{admiral}`.  Currently we test `{admiral}` against R Version 3.6.3 with a CRAN package snapshot from 2020-02-29, R Version 4.0 with a CRAN package snapshot from 2021-03-31 and the latest R version with the latest snapshots of packages.  You can view this workflow on our [GitHub Repository](https://github.com/pharmaverse/admiral/blob/main/.github/workflows/R-CMD-check.yml).
```
Comment (bms63) [inline vignettes/faq.Rmd:53]:
```suggestion
* GitHub allows us through the Actions/Workflows to test `{admiral}` under several versions of R as well as several versions of dependent R packages needed for `{admiral}`.  Currently we test `{admiral}` against R Version 3.6.3 with a CRAN package snapshot from 2020-02-29, R Version 4.0 with a CRAN package snapshot from 2021-03-31 and the latest R version with the latest snapshots of packages.  You can view this workflow and others on our [admiralci GitHub Repository](https://github.com/pharmaverse/admiralci).
```

File: NEWS.md [hunk 5]
```diff
@@ -1,3 +1,8 @@
+# admiral 0.8.1
+
+- `derive_var_extreme_dt()` and `derive_var_extreme_dtm()` were updated such
+that source observations where the date is `NA` are excluded (#1419).
```
Comment (thomas-neitmann) [inline NEWS.md:4]:
```suggestion
- `derive_var_extreme_dt()` and `derive_var_extreme_dtm()` were updated such
that source observations where the date is `NA` are excluded (#1419)
```

We don't use full stops unless it's a multi-sentence paragraph.

File: tests/testthat/test-derive_var_extreme_date.R [hunk 6]
```diff
@@ -97,9 +99,33 @@ test_that("derive_var_extreme_dt Test 2: LSTALVDT is derived for Date class as w
   )
 })
 
+## Test 3: NA dates are excluded ----
+test_that("derive_var_extreme_dt Test 3: NA dates are excluded", {
```
Comment (thomas-neitmann) [inline tests/testthat/test-derive_var_extreme_date.R:103]:
```suggestion
test_that("derive_var_extreme_dt Test 3: `NA` dates are excluded", {
```

File: tests/testthat/test-derive_var_extreme_date.R [hunk 7]
```diff
@@ -97,9 +99,33 @@ test_that("derive_var_extreme_dt Test 2: LSTALVDT is derived for Date class as w
...
+    compare = actual_output,
+    keys = c("USUBJID")
+  )
+})
+
 # derive_var_extreme_dtm ----
-## Test 3: LSTALVDTM and traceability variables are derived ----
-test_that("derive_var_extreme_dtm Test 3: LSTALVDTM and traceability variables are derived", {
+## Test 4: LSTALVDTM and traceability variables are derived ----
+test_that("derive_var_extreme_dtm Test 4: LSTALVDTM and traceability variables are derived", {
```
Comment (thomas-neitmann) [inline tests/testthat/test-derive_var_extreme_date.R:128]:
```suggestion
test_that("derive_var_extreme_dtm Test 4: `LSTALVDTM` and traceability variables are derived", {
```

File: tests/testthat/test-derive_var_extreme_date.R [hunk 8]
```diff
@@ -164,8 +190,8 @@ test_that("derive_var_extreme_dtm Test 3: LSTALVDTM and traceability variables a
   )
 })
 
-## Test 4: error is issued if DTC variable is specified ----
-test_that("derive_var_extreme_dtm Test 4: error is issued if DTC variable is specified", {
+## Test 5: error is issued if DTC variable is specified ----
+test_that("derive_var_extreme_dtm Test 5: error is issued if DTC variable is specified", {
```
Comment (thomas-neitmann) [inline tests/testthat/test-derive_var_extreme_date.R:194]:
```suggestion
test_that("derive_var_extreme_dtm Test 5: error is issued if `--DTC` variable is specified", {
```

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]

**Comment by bms63** [comment]
@thomas-neitmann Looks good to me.   Can you merge in as I created the PR

**Review by thomas-neitmann — APPROVED** [review]

**Pull request merged**